### PR TITLE
Add React frontend page with API endpoint that communicates with Node server

### PR DESCRIPTION
**Pull request by gazijarin**
Added a new client folder for all our client needs.

### CODE REVIEW

**Review by AbhiKapoor55 — APPROVED** [review]

File: webapp/app/client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (DemetreJou) [inline webapp/app/client/.gitignore:1]: This should probably be combine with the root level .gitignore

File: webapp/app/client/public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (DemetreJou) [inline webapp/app/client/public/robots.txt:3]: Should probably disallow all to default

File: webapp/app/server/server.js [hunk 3]
```diff
@@ -1,46 +1,51 @@
-const express = require('express')
-const app = express()
-const config = require('./environment')
-port = config.webapp.port
-const axios = require('axios')
-const { v4: uuidv4 } = require('uuid')
+const express = require("express");
```
Comment (DemetreJou) [inline webapp/app/server/server.js:1]: Ty for linting this

File: webapp/npm-debug.log [hunk 4]
```diff
@@ -0,0 +1,2275 @@
+0 info it worked if it ends with ok
```
Comment (DemetreJou) [inline webapp/npm-debug.log:1]: This probably shouldn't be commited?

**Pull request merged**